language/en-GB/tpl_cassiopeia.ini

```
@@ -9,5 +9,6 @@ TPL_CASSIOPEIA_FLUID_LABEL="Fluid Layout"
 TPL_CASSIOPEIA_LOGO_LABEL="Logo"
 TPL_CASSIOPEIA_STATIC="Static"
 TPL_CASSIOPEIA_STICKY_LABEL="Sticky Header"
+TPL_CASSIOPEIA_BACKTOP_LABEL="Back-to-top button"
```

```suggestion
TPL_CASSIOPEIA_BACKTOP_LABEL="Back-to-top Button"
```

To follow the style guidelines

---

Changed on new commit

---

Changed with new commit